tools/run_tests/sanity/sanity_tests.yaml

```diff
@@ -24,6 +24,9 @@
 - script: tools/distrib/check_pytype.sh
 - script: tools/distrib/clang_format_code.sh
 - script: tools/distrib/clang_tidy_code.sh
+  # ClangTidy needs to run exclusively because it runs bazel making changes
+  # and it needs many CPUs to parse all source files to be diagnosed.
+  cpu_cost: 1000
```

Comment:
What is the average clang-tidy check duration? We want to make our sanity check run as fast as possible.

Reply:
This couldn't be faster by design. It should compile all C++ header and source files to draw a conclusion. It usually takes 7-8 minutes on my gLinux desktop.